great-idea/index.html

```diff
@@ -17,6 +17,89 @@
   <body>
 
     <!-- Copy and paste your HTML from the first UI project here -->
+
+    <div class = "container">
+      <header>
+        <div class = "header-top-row">
+          <nav>
+            <a href = "services.html">Services</a>
+            <a href = "#">Product</a>
+            <a href = "#">Vision</a>
+            <a href = "#">Features</a>
+            <a href = "#">About</a>
+            <a href = "#">Contact</a>
+          </nav>
+
+          <a href="index.html"><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."></a>
+        </div>
```

Comment:
It might have been a little easier for your styling to close your header here and start a new section for the CTA.